### PR TITLE
Balder/now generates extra fields also for streaming

### PR DESCRIPTION
**Pull request by baldersheim**
@jonmv @geirst @vekterli @toregge  PR

### CODE REVIEW

**Review by jonmv — COMMENTED** [review]
Sweet

**Review by vekterli — COMMENTED** [review]
:+1:

File: tests/search/schemachanges/schemachanges_need_refeed_reconfig.rb [hunk 1]
```diff
@@ -105,19 +105,17 @@ def test_need_refeed_after_indexing_mode_change
 
     # Feed should be accepted
     feed_output = feed(:file => @test_dir + "feed.1.xml", :timeout => 20, :exceptiononfailure => false)
-    # Search & docsum should still work, but only the document fed while indexing mode was "index" (ELASTIC) has had indexing script run
-    assert_result("sddocname:test&nocache", @test_dir + "result.1.json")
-    assert_hitcount("f1:b&nocache", 1)          # No index for old document
-    assert_hitcount("f3:%3E29&nocache", 2)      # But attributes work
+    # Search & docsum should still work also for genrated fields
```
Comment (geirst) [inline tests/search/schemachanges/schemachanges_need_refeed_reconfig.rb:108]: genrated -> generated

**Review by geirst — COMMENTED** [review]
👍

**Pull request merged**